WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -1331,8 +1332,18 @@ public static void viewPagesInNewStack(Context context) {
         context.startActivity(intent);
     }
 
-    public static void showBackupDownload(Activity activity) {
+    public static void showBackupDownloadForResult(
```

Comment:
Nitpick 🔍 : We usually don't put parameter-per-line in java classes. We might want to change this to:
```suggestion
    public static void showBackupDownloadForResult(Activity activity, SiteModel site, String activityId,
                                                   int resultCode) {
```
Wdyt?

Reply:
> We usually don't put parameter-per-line in java classes

I did not know this. Thanks

Reply:
in 4fabf1b